webcompat/static/js/lib/homepage.js

```diff
@@ -63,6 +64,39 @@ function HomePage() {
     });
   };
 
+  this.navbarHandler = function() {
+    var $navbar = $(".navigation");
+    var navbarHeight = $navbar.outerHeight();
+    var lastScrollY = window.pageYOffset;
+    var scrollTimeout = null;
+    var isScrolling = false;
+    $(window).on("scroll", function() {
+      isScrolling = true;
+      clearTimeout(scrollTimeout);
+      scrollTimeout = window.setTimeout(function() {
+        isScrolling = false;
+      }, 500);
+    });
+    window.setInterval(function() {
+      if (!isScrolling) return;
```

Comment:
nit: can we add opening and closing braces and put the return on newline?

Reply:
Sure.